collectors/scm/gitlab/src/main/java/com/capitalone/dashboard/gitlab/GitlabUrlUtility.java

```diff
@@ -73,17 +87,155 @@ public URI buildApiUrl(GitlabGitRepo repo, boolean firstRun, int resultsPerPage)
 				.queryParam(PER_PAGE_QUERY_PARAM_KEY, resultsPerPage)
 				.build(true).toUri();
 
-		LOG.info("---> Gitlab URI: " + uri.toString());
+		LOG.info("---> Gitlab commits URI: " + uri.toString());
 		return uri;
     }
 
-    private String getProtocol() {
-        return StringUtils.isBlank(gitlabSettings.getProtocol()) ? DEFAULT_PROTOCOL : gitlabSettings.getProtocol();
-    }
+	public URI buildIssuesApiUrl(GitlabGitRepo repo, boolean firstRun, int resultsPerPage) {
+		String repoUrl = repo.getRepoUrl();
+        if (repoUrl.endsWith(GIT_EXTENSION)) {
+            repoUrl = StringUtils.removeEnd(repoUrl, GIT_EXTENSION);
+        }
 
-    private String getApiVersion() {
-        return gitlabSettings.getApiVersion() == 3 ? V3 : V4;
-    }
+		String apiVersion = getApiVersion();
+		String protocol = getProtocol();
+		String repoName = getRepoName(repoUrl);
+		String host = getRepoHost();
+		String date = getSinceDate(repo, firstRun);
+
+		UriComponentsBuilder builder = UriComponentsBuilder.newInstance();
+
+		if (StringUtils.isNotBlank(gitlabSettings.getPort())) {
+			builder.port(gitlabSettings.getPort());
+		}
+
+		URI uri = builder.scheme(protocol).host(host).path(gitlabSettings.getPath()).pathSegment(SEGMENT_API)
+				.pathSegment(apiVersion).pathSegment(PROJECTS_SEGMENT).path(repoName).path(ISSUES_API)
+				.queryParam(SCOPE_QUERY_PARAM_KEY, "all").queryParam(CREATED_AFTER_QUERY_PARAM_KEY, date)
+				.queryParam(PER_PAGE_QUERY_PARAM_KEY, resultsPerPage)
+				.queryParam(ORDER_BY_QUERY_PARAM_KEY, "updated_at").queryParam(SORT_QUERY_PARAM_KEY, "desc")
+				.build(true).toUri();
+
+		LOG.info("---> Gitlab issues URI: " + uri.toString());
+		return uri;
+	}
+
+	public URI buildMergeRequestsApiUrl(GitlabGitRepo repo, String status, boolean firstRun,
+			int resultsPerPage) {
+		String repoUrl = repo.getRepoUrl();
+        if (repoUrl.endsWith(GIT_EXTENSION)) {
```

Comment:
Formatting

Reply:
Changed to space!

https://github.com/Hygieia/Hygieia/pull/2755/commits/3cbb00eff8b1daf70388ea69d7c1d03eb0682045#diff-a1c5a6b5d72aaed0621398f4d715f22bR131